api/explorer.go

```diff
@@ -188,13 +194,18 @@ func (api *API) buildExplorerBlock(height types.BlockHeight, block types.Block)
 		panic("incorrect request to buildExplorerBlock - block does not exist")
 	}
 
-	return ExplorerBlock{
+	b := ExplorerBlock{
 		MinerPayoutIDs: mpoids,
 		Transactions:   etxns,
 		RawBlock:       block,
+		HexBlockID:     hex.EncodeToString(encoding.Marshal(block.ID())),
 
 		BlockFacts: facts,
 	}
+	if hexBlockEnable {
+		b.HexBlock = hex.EncodeToString(encoding.Marshal(block))
```

Comment:
Could you implement something like the following in `types/block.go`?
```
// String returns a hex encoded marshalled Block.
func (b Block) String() string {                                                                                                                                                                                  
    return hex.EncodeToString(encoding.Marshal(b))                                                                                                                                                                                     
} 
```
That way you can simply do `b.HexBlock = block.String()` here.